ape_safe/_cli.py

```diff
@@ -45,9 +46,8 @@ def _list(cli_ctx, network):
         try:
             extras.append(f"version: '{account.version}'")
         except ChainError:
-            cli_ctx.logger.warning(
-                f"Not connected to the network that {account.address} is deployed"
-            )
+            # Not connected to the network where safe is deployed
```

Comment:
Agree on this change not to make it a warning, but maybe instead of `(1.3.0)` can display `(not connected)`